### ISSUE TITLE
Disarm after alarm is triggered

### ISSUE THREAD

**Issue by scotteby** [dustindclark/homebridge-envisalink#20]
I just noticed that I'm not able to disarm the panel from the home app after the alarm is triggered. 

When I click "Disarm" in the Home app, the app alarm status changes to Disarmed, but the actual alarm panel is not changed. I have to go and enter the code at the panel to turn it off.

Has anyone else seen this behavior? It looks like the Home app doesn't send the disarm command in this situation.

**Comment by ghost** [comment]
Do you have the right passcode in your config file?

Sent from my iPhone

On Aug 5, 2017, at 10:33 AM, echobravo <notifications@github.com<mailto:notifications@github.com>> wrote:


I just noticed that I'm not able to disarm the panel from the home app after the alarm is triggered.

When I click "Disarm" in the Home app, the app alarm status changes to Disarmed, but the actual alarm panel is not changed. I have to go and enter the code at the panel to turn it off.

Has anyone else seen this behavior? It looks like the Home app doesn't send the disarm command in this situation.

**Comment by scotteby** [comment]
Yes, I should have mentioned that I can disarm the system normally from the
app.

It's the only the case where the system is armed, and a sensor such as a
door is opened causing the alarm to be triggered. I can't disarm the system
from the app in this state.

Looking at the console, it looks like the disarm command is not being sent
to homebridge, but hoping someone else can confirm this to make sure it
isn't a configuration issue on my end.

**Comment by erfind** [comment]
Experiencing the same issue. After it is triggered there isn’t a way to disarm it while the alarm is triggered.

**Comment by dstapleton92** [comment]
Hi! I'm currently working on a re-write of this plugin that will resolve this issue. About 90% of the way done.

**Comment by erfind** [comment]
Thank you! 😃

**Comment by adrianmihalko** [comment]
@dstapleton92 

What do you think, is my problem related to this issue (Alarm status is not updated in Homekit)?

https://github.com/dustindclark/homebridge-envisalink/issues/37

Is your version ready? 🤘

**Comment by adrianmihalko** [comment]
@dstapleton92 any progress on your version?

**Comment by dstapleton92** [comment]
Hi @adrianmihalko  I think it's done. I haven't had any time to dedicate for testing, so I'm hesitant to open a Pull Request just yet. If you'd like to test it (please put it through the wringer and try to "break" it), you can find it [here](https://github.com/dstapleton92/homebridge-envisalink). It's written in ES6 and uses Babel to transpile. Make sure you read the note for developers I added at the bottom of the README file. (Mainly, `npm install`, then `npm run build` to create the compiled JS files that will work with Node/Homebridge.

**Comment by adrianmihalko** [comment]
What a good news, I don't promise that I test it today, but in a few days I will do it. 👍😀

**Comment by adrianmihalko** [comment]
I can confirm, your version is work great. Every partition, like it should. I keep it testing in the coming days. If I find something, I'll report it. If not, you can open a pull request. 

Great job from you, thank you very much. 💯👍


---

There was a little glitch on first run, status of partitions was not recognized correctly, but after one arming everything started working normally. I tested everything: arming/disarming manually, arming/disarming from Homebridge. Everything works great.

```
ay 06 00:32:31 ubuntu homebridge[32143]: [5/6/2018, 12:32:31 AM] [Envisalink] Initializing Envisalink platform...
May 06 00:32:31 ubuntu homebridge[32143]: [5/6/2018, 12:32:31 AM] [Envisalink] Configuring Envisalink platform,  Host: 192.168.1.54, port: undefi
May 06 00:32:31 ubuntu homebridge[32143]: [5/6/2018, 12:32:31 AM] [Envisalink] Starting node alarm proxy...
May 06 00:32:31 ubuntu homebridge[32143]: [5/6/2018, 12:32:31 AM] [Envisalink] Zones: 10
May 06 00:32:31 ubuntu homebridge[32143]: [5/6/2018, 12:32:31 AM] [Envisalink] User Programs: null
May 06 00:32:31 ubuntu homebridge[32143]: [5/6/2018, 12:32:31 AM] [Envisalink] Node alarm proxy started.  Listening for connections at: 0.0.0.0:4
May 06 00:32:31 ubuntu homebridge[32143]: [5/6/2018, 12:32:31 AM] [Envisalink] Initializing platform accessory 'Uzlet'...
May 06 00:32:31 ubuntu homebridge[32143]: [5/6/2018, 12:32:31 AM] [Envisalink] Initializing platform accessory 'Lakas'...
May 06 00:32:31 ubuntu homebridge[32143]: [5/6/2018, 12:32:31 AM] [Envisalink] Initializing platform accessory 'Lakás/ajtó'...
May 06 00:32:31 ubuntu homebridge[32143]: [5/6/2018, 12:32:31 AM] [Envisalink] Initializing platform accessory 'Lakás/mozgás'...
May 06 00:32:31 ubuntu homebridge[32143]: [5/6/2018, 12:32:31 AM] [Envisalink] Initializing platform accessory 'Shop - front door'...
May 06 00:32:31 ubuntu homebridge[32143]: [5/6/2018, 12:32:31 AM] [Envisalink] Initializing platform accessory 'Shop - motion'...
May 06 00:32:31 ubuntu homebridge[32143]: [5/6/2018, 12:32:31 AM] [Envisalink] Initializing platform accessory 'Warehouse'...
li

May 06 00:32:31 ubuntu homebridge[32143]: [5/6/2018, 12:32:31 AM] [Envisalink] Initializing platform accessory 'Servis/mozgás'...
May 06 00:32:31 ubuntu homebridge[32143]: [5/6/2018, 12:32:31 AM] [Envisalink] Initializing platform accessory 'Servis/ajtó'...
May 06 00:32:31 ubuntu homebridge[32143]: [5/6/2018, 12:32:31 AM] Loading 0 accessories...

May 06 00:32:32 ubuntu homebridge[32143]: server bound
May 06 00:32:32 ubuntu homebridge[32143]: actual connected
May 06 00:32:32 ubuntu homebridge[32143]: Login Request Response: 3
May 06 00:32:32 ubuntu homebridge[32143]: login requested... sending response...
May 06 00:32:32 ubuntu homebridge[32143]: sendcommand 005user54
May 06 00:32:32 ubuntu homebridge[32143]: Login Request Response: 1
May 06 00:32:32 ubuntu homebridge[32143]: successfully logged in!  getting current data...
May 06 00:32:32 ubuntu homebridge[32143]: sendcommand 00191
May 06 00:32:32 ubuntu homebridge[32143]: Command 001 Acknowledged

May 06 00:32:32 ubuntu homebridge[32143]: Zone 001 is restored
May 06 00:32:32 ubuntu homebridge[32143]: [5/6/2018, 12:32:32 AM] [Envisalink] System status changed to:  { zone: { '1': { send: 'restore', name:
May 06 00:32:32 ubuntu homebridge[32143]:   partition: {},
May 06 00:32:32 ubuntu homebridge[32143]:   user: {} }
May 06 00:32:32 ubuntu homebridge[32143]: Zone 002 is restored
May 06 00:32:32 ubuntu homebridge[32143]: [5/6/2018, 12:32:32 AM] [Envisalink] System status changed to:  { zone:
May 06 00:32:32 ubuntu homebridge[32143]:    { '1': { send: 'restore', name: 'Zone Alarm Restore', code: '61000128' },
May 06 00:32:32 ubuntu homebridge[32143]:      '2': { send: 'restore', name: 'Zone Alarm Restore', code: '61000229' } },
May 06 00:32:32 ubuntu homebridge[32143]:   partition: {},
May 06 00:32:32 ubuntu homebridge[32143]:   user: {} }
May 06 00:32:32 ubuntu homebridge[32143]: Zone 003 is restored
May 06 00:32:32 ubuntu homebridge[32143]: [5/6/2018, 12:32:32 AM] [Envisalink] System status changed to:  { zone:
May 06 00:32:32 ubuntu homebridge[32143]:    { '1': { send: 'restore', name: 'Zone Alarm Restore', code: '61000128' },
May 06 00:32:32 ubuntu homebridge[32143]:      '2': { send: 'restore', name: 'Zone Alarm Restore', code: '61000229' },
May 06 00:32:32 ubuntu homebridge[32143]:      '3': { send: 'restore', name: 'Zone Alarm Restore', code: '6100032A' } },
May 06 00:32:32 ubuntu homebridge[32143]:   partition: {},
May 06 00:32:32 ubuntu homebridge[32143]:   user: {} }


May 06 00:32:32 ubuntu homebridge[32143]: Zone 004 is restored
May 06 00:32:32 ubuntu homebridge[32143]: [5/6/2018, 12:32:32 AM] [Envisalink] System status changed to:  { zone:
May 06 00:32:32 ubuntu homebridge[32143]:    { '1': { send: 'restore', name: 'Zone Alarm Restore', code: '61000128' },
May 06 00:32:32 ubuntu homebridge[32143]:      '2': { send: 'restore', name: 'Zone Alarm Restore', code: '61000229' },
May 06 00:32:32 ubuntu homebridge[32143]:      '3': { send: 'restore', name: 'Zone Alarm Restore', code: '6100032A' },
May 06 00:32:32 ubuntu homebridge[32143]:      '4': { send: 'restore', name: 'Zone Alarm Restore', code: '6100042B' } },
May 06 00:32:32 ubuntu homebridge[32143]:   partition: {},
May 06 00:32:32 ubuntu homebridge[32143]:   user: {} }
May 06 00:32:32 ubuntu homebridge[32143]: Zone 005 is restored
May 06 00:32:32 ubuntu homebridge[32143]: [5/6/2018, 12:32:32 AM] [Envisalink] System status changed to:  { zone:
May 06 00:32:32 ubuntu homebridge[32143]:    { '1': { send: 'restore', name: 'Zone Alarm Restore', code: '61000128' },
May 06 00:32:32 ubuntu homebridge[32143]:      '2': { send: 'restore', name: 'Zone Alarm Restore', code: '61000229' },
May 06 00:32:32 ubuntu homebridge[32143]:      '3': { send: 'restore', name: 'Zone Alarm Restore', code: '6100032A' },
May 06 00:32:32 ubuntu homebridge[32143]:      '4': { send: 'restore', name: 'Zone Alarm Restore', code: '6100042B' },
May 06 00:32:32 ubuntu homebridge[32143]:      '5': { send: 'restore', name: 'Zone Alarm Restore', code: '6100052C' } },
May 06 00:32:32 ubuntu homebridge[32143]:   partition: {},
May 06 00:32:32 ubuntu homebridge[32143]:   user: {} }
May 06 00:32:32 ubuntu homebridge[32143]: Zone 006 is restored
May 06 00:32:32 ubuntu homebridge[32143]: [5/6/2018, 12:32:32 AM] [Envisalink] System status changed to:  { zone:
May 06 00:32:32 ubuntu homebridge[32143]:    { '1': { send: 'restore', name: 'Zone Alarm Restore', code: '61000128' },
May 06 00:32:32 ubuntu homebridge[32143]:      '2': { send: 'restore', name: 'Zone Alarm Restore', code: '61000229' },
May 06 00:32:32 ubuntu homebridge[32143]:      '3': { send: 'restore', name: 'Zone Alarm Restore', code: '6100032A' },
May 06 00:32:32 ubuntu homebridge[32143]:      '4': { send: 'restore', name: 'Zone Alarm Restore', code: '6100042B' },
May 06 00:32:32 ubuntu homebridge[32143]:      '5': { send: 'restore', name: 'Zone Alarm Restore', code: '6100052C' },
May 06 00:32:32 ubuntu homebridge[32143]:      '6': { send: 'restore', name: 'Zone Alarm Restore', code: '6100062D' } },
May 06 00:32:32 ubuntu homebridge[32143]:   partition: {},
May 06 00:32:32 ubuntu homebridge[32143]:   user: {} }
M

May 06 00:32:32 ubuntu homebridge[32143]: Zone 007 is restored
May 06 00:32:32 ubuntu homebridge[32143]: [5/6/2018, 12:32:32 AM] [Envisalink] System status changed to:  { zone:
May 06 00:32:32 ubuntu homebridge[32143]:    { '1': { send: 'restore', name: 'Zone Alarm Restore', code: '61000128' },
May 06 00:32:32 ubuntu homebridge[32143]:      '2': { send: 'restore', name: 'Zone Alarm Restore', code: '61000229' },
May 06 00:32:32 ubuntu homebridge[32143]:      '3': { send: 'restore', name: 'Zone Alarm Restore', code: '6100032A' },
May 06 00:32:32 ubuntu homebridge[32143]:      '4': { send: 'restore', name: 'Zone Alarm Restore', code: '6100042B' },
May 06 00:32:32 ubuntu homebridge[32143]:      '5': { send: 'restore', name: 'Zone Alarm Restore', code: '6100052C' },
May 06 00:32:32 ubuntu homebridge[32143]:      '6': { send: 'restore', name: 'Zone Alarm Restore', code: '6100062D' },
May 06 00:32:32 ubuntu homebridge[32143]:      '7': { send: 'restore', name: 'Zone Alarm Restore', code: '6100072E' } },
May 06 00:32:32 ubuntu homebridge[32143]:   partition: {},
May 06 00:32:32 ubuntu homebridge[32143]:   user: {} }
May 06 00:32:32 ubuntu homebridge[32143]: Zone 008 is restored
May 06 00:32:32 ubuntu homebridge[32143]: [5/6/2018, 12:32:32 AM] [Envisalink] System status changed to:  { zone:
May 06 00:32:32 ubuntu homebridge[32143]:    { '1': { send: 'restore', name: 'Zone Alarm Restore', code: '61000128' },
May 06 00:32:32 ubuntu homebridge[32143]:      '2': { send: 'restore', name: 'Zone Alarm Restore', code: '61000229' },
May 06 00:32:32 ubuntu homebridge[32143]:      '3': { send: 'restore', name: 'Zone Alarm Restore', code: '6100032A' },
May 06 00:32:32 ubuntu homebridge[32143]:      '4': { send: 'restore', name: 'Zone Alarm Restore', code: '6100042B' },
May 06 00:32:32 ubuntu homebridge[32143]:      '5': { send: 'restore', name: 'Zone Alarm Restore', code: '6100052C' },
May 06 00:32:32 ubuntu homebridge[32143]:      '6': { send: 'restore', name: 'Zone Alarm Restore', code: '6100062D' },
May 06 00:32:32 ubuntu homebridge[32143]:      '7': { send: 'restore', name: 'Zone Alarm Restore', code: '6100072E' },
May 06 00:32:32 ubuntu homebridge[32143]:      '8': { send: 'restore', name: 'Zone Alarm Restore', code: '6100082F' } },
May 06 00:32:32 ubuntu homebridge[32143]:   partition: {},
May 06 00:32:32 ubuntu homebridge[32143]:   user: {} }


May 06 00:32:32 ubuntu homebridge[32143]: Zone 009 is restored
May 06 00:32:32 ubuntu homebridge[32143]: [5/6/2018, 12:32:32 AM] [Envisalink] System status changed to:  { zone:
May 06 00:32:32 ubuntu homebridge[32143]:    { '1': { send: 'restore', name: 'Zone Alarm Restore', code: '61000128' },
May 06 00:32:32 ubuntu homebridge[32143]:      '2': { send: 'restore', name: 'Zone Alarm Restore', code: '61000229' },
May 06 00:32:32 ubuntu homebridge[32143]:      '3': { send: 'restore', name: 'Zone Alarm Restore', code: '6100032A' },
May 06 00:32:32 ubuntu homebridge[32143]:      '4': { send: 'restore', name: 'Zone Alarm Restore', code: '6100042B' },
May 06 00:32:32 ubuntu homebridge[32143]:      '5': { send: 'restore', name: 'Zone Alarm Restore', code: '6100052C' },
May 06 00:32:32 ubuntu homebridge[32143]:      '6': { send: 'restore', name: 'Zone Alarm Restore', code: '6100062D' },
May 06 00:32:32 ubuntu homebridge[32143]:      '7': { send: 'restore', name: 'Zone Alarm Restore', code: '6100072E' },
May 06 00:32:32 ubuntu homebridge[32143]:      '8': { send: 'restore', name: 'Zone Alarm Restore', code: '6100082F' },
May 06 00:32:32 ubuntu homebridge[32143]:      '9': { send: 'restore', name: 'Zone Alarm Restore', code: '61000930' } },
May 06 00:32:32 ubuntu homebridge[32143]:   partition: {},
May 06 00:32:32 ubuntu homebridge[32143]:   user: {} }
May 06 00:32:32 ubuntu homebridge[32143]: Zone 010 is open
May 06 00:32:32 ubuntu homebridge[32143]: [5/6/2018, 12:32:32 AM] [Envisalink] System status changed to:  { zone:
May 06 00:32:32 ubuntu homebridge[32143]:    { '1': { send: 'restore', name: 'Zone Alarm Restore', code: '61000128' },
May 06 00:32:32 ubuntu homebridge[32143]:      '2': { send: 'restore', name: 'Zone Alarm Restore', code: '61000229' },
May 06 00:32:32 ubuntu homebridge[32143]:      '3': { send: 'restore', name: 'Zone Alarm Restore', code: '6100032A' },
May 06 00:32:32 ubuntu homebridge[32143]:      '4': { send: 'restore', name: 'Zone Alarm Restore', code: '6100042B' },
May 06 00:32:32 ubuntu homebridge[32143]:      '5': { send: 'restore', name: 'Zone Alarm Restore', code: '6100052C' },
May 06 00:32:32 ubuntu homebridge[32143]:      '6': { send: 'restore', name: 'Zone Alarm Restore', code: '6100062D' },
May 06 00:32:32 ubuntu homebridge[32143]:      '7': { send: 'restore', name: 'Zone Alarm Restore', code: '6100072E' },
May 06 00:32:32 ubuntu homebridge[32143]:      '8': { send: 'restore', name: 'Zone Alarm Restore', code: '6100082F' },
May 06 00:32:32 ubuntu homebridge[32143]:      '9': { send: 'restore', name: 'Zone Alarm Restore', code: '61000930' },
May 06 00:32:32 ubuntu homebridge[32143]:      '10': { send: 'open', name: 'Zone Open', code: '60901030' } },
May 06 00:32:32 ubuntu homebridge[32143]:   partition: {},
May 06 00:32:32 ubuntu homebridge[32143]:   user: {} }

May 06 00:32:32 ubuntu homebridge[32143]: Zone 011 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 012 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 013 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 014 is restored
May 06 00:32:32 ubuntu homebridge[32143]: [5/6/2018, 12:32:32 AM] [eDomoticz] Successfully connected to MQTT broker.
May 06 00:32:32 ubuntu homebridge[32143]: Zone 015 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 016 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 017 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 018 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 019 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 020 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 021 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 022 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 023 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 024 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 025 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 026 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 027 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 028 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 029 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 030 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 031 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 032 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 033 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 034 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 035 is restored


May 06 00:32:32 ubuntu homebridge[32143]: Zone 036 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 037 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 038 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 039 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 040 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 041 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 042 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 043 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 044 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 045 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 046 is restored
May 06 00:32:32 ubuntu homebridge[32143]: Zone 047 is restored
May 06 00:32:33 ubuntu homebridge[32143]: Zone 048 is restored
May 06 00:32:33 ubuntu homebridge[32143]: Zone 049 is restored
May 06 00:32:33 ubuntu homebridge[32143]: Zone 050 is restored
May 06 00:32:33 ubuntu homebridge[32143]: Zone 051 is restored
May 06 00:32:33 ubuntu homebridge[32143]: Zone 052 is restored
May 06 00:32:33 ubuntu homebridge[32143]: Zone 053 is restored
May 06 00:32:33 ubuntu homebridge[32143]: Zone 054 is restored

May 06 00:32:33 ubuntu homebridge[32143]: Zone 055 is restored
May 06 00:32:33 ubuntu homebridge[32143]: Zone 056 is restored
May 06 00:32:33 ubuntu homebridge[32143]: Zone 057 is restored
May 06 00:32:33 ubuntu homebridge[32143]: Zone 058 is restored
May 06 00:32:33 ubuntu homebridge[32143]: Zone 059 is restored
May 06 00:32:33 ubuntu homebridge[32143]: Zone 060 is restored
May 06 00:32:33 ubuntu homebridge[32143]: Zone 061 is restored
May 06 00:32:33 ubuntu homebridge[32143]: Zone 062 is restored
May 06 00:32:33 ubuntu homebridge[32143]: Zone 063 is restored
May 06 00:32:33 ubuntu homebridge[32143]: Zone 064 is restored
May 06 00:32:33 ubuntu homebridge[32143]: Partition 10 is Armed
May 06 00:32:33 ubuntu homebridge[32143]: [5/6/2018, 12:32:33 AM] [Envisalink] System status changed to:  { zone:
May 06 00:32:33 ubuntu homebridge[32143]:    { '1': { send: 'restore', name: 'Zone Alarm Restore', code: '61000128' },
May 06 00:32:33 ubuntu homebridge[32143]:      '2': { send: 'restore', name: 'Zone Alarm Restore', code: '61000229' },
May 06 00:32:33 ubuntu homebridge[32143]:      '3': { send: 'restore', name: 'Zone Alarm Restore', code: '6100032A' },
May 06 00:32:33 ubuntu homebridge[32143]:      '4': { send: 'restore', name: 'Zone Alarm Restore', code: '6100042B' },
May 06 00:32:33 ubuntu homebridge[32143]:      '5': { send: 'restore', name: 'Zone Alarm Restore', code: '6100052C' },
May 06 00:32:33 ubuntu homebridge[32143]:      '6': { send: 'restore', name: 'Zone Alarm Restore', code: '6100062D' },
May 06 00:32:33 ubuntu homebridge[32143]:      '7': { send: 'restore', name: 'Zone Alarm Restore', code: '6100072E' },
May 06 00:32:33 ubuntu homebridge[32143]:      '8': { send: 'restore', name: 'Zone Alarm Restore', code: '6100082F' },
May 06 00:32:33 ubuntu homebridge[32143]:      '9': { send: 'restore', name: 'Zone Alarm Restore', code: '61000930' },
May 06 00:32:33 ubuntu homebridge[32143]:      '10': { send: 'open', name: 'Zone Open', code: '60901030' } },
May 06 00:32:33 ubuntu homebridge[32143]:   partition: { '1': { send: 'armed', name: 'Partition Armed', code: '65210FE' } },
May 06 00:32:33 ubuntu homebridge[32143]:   user: {} }


May 06 00:32:33 ubuntu homebridge[32143]: [5/6/2018, 12:32:33 AM] [Envisalink] Setting state from EVL: {"name":"Partition Armed","pre":"Partition","b
May 06 00:32:33 ubuntu homebridge[32143]: Set system status on accessory Uzlet to undefined
May 06 00:32:33 ubuntu homebridge[32143]: Partition 2 is NOT Ready
May 06 00:32:33 ubuntu homebridge[32143]: [5/6/2018, 12:32:33 AM] [Envisalink] System status changed to:  { zone:
May 06 00:32:33 ubuntu homebridge[32143]:    { '1': { send: 'restore', name: 'Zone Alarm Restore', code: '61000128' },
May 06 00:32:33 ubuntu homebridge[32143]:      '2': { send: 'restore', name: 'Zone Alarm Restore', code: '61000229' },
May 06 00:32:33 ubuntu homebridge[32143]:      '3': { send: 'restore', name: 'Zone Alarm Restore', code: '6100032A' },
May 06 00:32:33 ubuntu homebridge[32143]:      '4': { send: 'restore', name: 'Zone Alarm Restore', code: '6100042B' },
May 06 00:32:33 ubuntu homebridge[32143]:      '5': { send: 'restore', name: 'Zone Alarm Restore', code: '6100052C' },
May 06 00:32:33 ubuntu homebridge[32143]:      '6': { send: 'restore', name: 'Zone Alarm Restore', code: '6100062D' },
May 06 00:32:33 ubuntu homebridge[32143]:      '7': { send: 'restore', name: 'Zone Alarm Restore', code: '6100072E' },
May 06 00:32:33 ubuntu homebridge[32143]:      '8': { send: 'restore', name: 'Zone Alarm Restore', code: '6100082F' },
May 06 00:32:33 ubuntu homebridge[32143]:      '9': { send: 'restore', name: 'Zone Alarm Restore', code: '61000930' },
May 06 00:32:33 ubuntu homebridge[32143]:      '10': { send: 'open', name: 'Zone Open', code: '60901030' } },
May 06 00:32:33 ubuntu homebridge[32143]:   partition:
May 06 00:32:33 ubuntu homebridge[32143]:    { '1': { send: 'armed', name: 'Partition Armed', code: '65210FE' },
May 06 00:32:33 ubuntu homebridge[32143]:      '2': { send: 'notready', name: 'Partition Not Ready', code: '6512CE' } },
May 06 00:32:33 ubuntu homebridge[32143]:   user: {} }
May 06 00:32:33 ubuntu homebridge[32143]: Set system status on accessory Uzlet to undefined
May 06 00:32:33 ubuntu homebridge[32143]: Set system status on accessory Lakas to undefined
May 06 00:32:33 ubuntu homebridge[32143]: Partition 3 is Busy
May 06 00:32:33 ubuntu homebridge[32143]: Partition 4 is Busy
May 06 00:32:33 ubuntu homebridge[32143]: Partition 5 is Busy


May 06 00:32:33 ubuntu homebridge[32143]: Partition 8 is Busy
May 06 00:32:33 ubuntu homebridge[32143]: Trouble LED of Partition 1 is OFF
May 06 00:32:34 ubuntu homebridge[32143]: Trouble LED of Partition 2 is OFF
May 06 00:32:34 ubuntu homebridge[32143]: Trouble LED of Partition 3 is OFF
May 06 00:32:34 ubuntu homebridge[32143]: Trouble LED of Partition 4 is OFF
May 06 00:32:34 ubuntu homebridge[32143]: Trouble LED of Partition 5 is OFF
May 06 00:32:34 ubuntu homebridge[32143]: Trouble LED of Partition 6 is OFF
May 06 00:32:34 ubuntu homebridge[32143]: Trouble LED of Partition 7 is OFF
May 06 00:32:34 ubuntu homebridge[32143]: Trouble LED of Partition 8 is OFF
May 06 00:32:34 ubuntu homebridge[32143]: Keypad LED State: 82
May 06 00:33:34 ubuntu homebridge[32143]: Command 000 Acknowledged

```

And one more error at install:

```
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

```

But maybe this is not a needed dependency?

**Comment by adrianmihalko** [comment]
Oh no, after some hours later it fails back to a weird state:
1. partition: disarming... (Newer complete)
2. Partition: Home (it was armed in away)

If I arm/disarm it works for a while.

**Comment by dustindclark** [comment]
Partition support fixed and published in v0.2.14.
